azext_edge/edge/params.py

```diff
@@ -1184,3 +1184,47 @@ def load_iotops_arguments(self, _):
             help="Fully qualified role definition Id in the following format: "
             "/subscriptions/{subscriptionId}/providers/Microsoft.Authorization/roleDefinitions/{roleId}",
         )
+
+    with self.argument_context("iot ops connector opcua trust") as context:
+        context.argument(
+            "instance_name",
+            options_list=["--instance", "-i"],
+            help="IoT Operations instance name.",
```

Comment:
We use `--name` in other instance commands to refer to the instance name, is there another param conflicting with that? I think it's a good idea to remain consistent with other commands